### DISCUSSION TITLE
Can Anyone help on this contract not found error in forge create ?

### DISCUSSION THREAD

**Discussion by vasanthmarimuth** [Cyfrin/foundry-full-course-cu#2474] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/fddd8036-d32b-437d-94b3-2a98f07a0dc5)
I start anvil and take a needed private keys and rpc url then stopped the anvil so that only it is possible to run the commands on terminal  ( is it necessary to run anvil in the background while deploy the smartcontract ) but it shows an error that contract not found. I already create a SimpleStorage.sol contract in src folder. I compiled too but I cannot rectify this error, kindly someone help me on this

**Comment by EngrPips** [comment, marked as answer]
Hello @vasanthmarimuth, You will need to be running `Anvil` in a separate terminal and then open another terminal to run all of your deployment and interaction commands. You can click on the icon I circled in the image below to open a new separate terminal.

<img width="1440" alt="Screenshot 2024-09-14 at 20 44 03" src="https://github.com/user-attachments/assets/246461d3-812c-443a-bec9-9f2194b05ef5">

**Comment by vasanthmarimuth** [comment]
Thanks, @EngrPips! It’s working great, and I’m excited to dive deeper and learn more.

**Reply by EngrPips** [reply]
I am glad it is working now. keep crushing it.
